Gems/Atom/Tools/AtomToolsFramework/Code/Source/Util/MaterialPropertyUtil.cpp

```diff
@@ -59,7 +59,7 @@ namespace AtomToolsFramework
         }
 
         AZ_Assert(false, "Attempting to convert an unsupported property type.");
-        return AtomToolsFramework::DynamicPropertyType::Invalid;
+        return AtomToolsFramework::DynamicPropertyType::Unspecified;
```

Comment:
I wonder if the change of the Invalid enum indicates some special handling for unspecified type, if so, the assert can probably be downgrade to warning.